### ISSUE TITLE
[HDIVSCode][Accessibility][Mac]Press "Control+Alt+Arrow key", the result table cell cannot be selected as required.

### ISSUE THREAD

**Issue by v-supupp** [microsoft/vscode#81883]
Environment Details:
Install build in attachment.

Visual studio code version:1.34.0​
OS: Mac

Repro Steps: 
1. Install build in attachment.
2. Open any python query and link with the cluster using proper credentials
3. Select the code and press shift + F10
4. Navigate till “HDInteractive: Hive Interactive” and press enter.
5. Navigate to Table section in the Result pane.
6. Keep pressing CTRL+ALT+RIGHT.

Actual Result: 
The cell cannot be selected as required. 

Expected Result: 
Press Control + Alt + Arrow to select cells as needed.

**Comment by Tyriar** [comment]
Please update to the latest version of VS Code, also all seems related to some extension?
